.github/labeler.yml

```diff
@@ -74,6 +74,13 @@
         - lib/systems/flake-systems.nix
         - nixos/modules/config/nix-flakes.nix
 
+"6.topic: flutter":
+  - any:
+    - changed-files:
+      - any-glob-to-any-file:
+        - pkgs/build-support/flutter/*.nix
+        - pkgs/development/compilers/flutter/**/*.nix
```

Comment:
(So ​​why are we restricting nix here? Maybe there are only nix files now, but not necessarily in the future?)

Reply:
Yeah, only 1